tools/wpt/tests/test_testfiles.py

```diff
@@ -11,3 +12,32 @@ def test_getrevish_kwarg():
 def test_getrevish_implicit():
     with patch("tools.wpt.testfiles.branch_point", return_value=u"base"):
         assert testfiles.get_revish() == b"base..HEAD"
+
+
+def test_affected_testfiles():
+    manifest_json = {
+        "items": {
+            "crashtest": {
+                "a/b/c/foo-crash.html": [
+                    [ "a/b/c/foo-crash.html", {} ],
+                ]
+            }
+        },
+        "paths": {
+            "a/b/c/foo-crash.html": [
+                "acdefgh123456",
+                "crashtest",
+            ]
+        },
+        "url_base": "/",
+        "version": 7,
+    }
+    mfst = manifest.Manifest.from_json("/", manifest_json)
```

Comment:
I am very interested in a better name than 'mfst' -_-

Reply:
```suggestion
    manifest = Manifest.from_json("/", manifest_json)
```